tests/test_worker.py

```diff
@@ -19,7 +19,7 @@
 
 try:
     import uvloop
-except ImportError:
+except ImportError:  # pragma: no cover
```

Comment:
I think this should be covered by pypy, so it shouldn't be ignored. We just need pypy to stop segfaulting half the time....

Reply:
Sure, but it's seems this case already covered by the code anyway

```
PARAMS = [AsyncioWorker]
if uvloop is not None:
    PARAMS.append(UvloopWorker)
```

And then tests checks both workers.

Reply:
I don't see that as a reason to skip coverage here?

Reply:
Cause it looks redundant here - case with `uvloop is None` already covered anyway

But if you are against it, then I can remove it `¯\_(ツ)_/¯`

```suggestion
except ImportError:
```

Reply:
Yep, I'm just unsure why redundancy would be a reason to ignore coverage (otherwise, we'd surely ignore 90% of the code...). The main reason to ignore code is that we know it'll never be run (primarily typing code). Another reason recently is some retry logic within a test used to make the test more stable, but the coverage could vary between test runs and confuse results.